templates/web/bristol/around/intro.html

```diff
@@ -1,2 +1,2 @@
 <h1>Report a problem</h1>
-<p>Report a problem on a street, car park, park, green space or harbourside area in Bristol or look at and discuss problems that have already been reported.</p>
+<h2 style="font-size:1em">Report a problem on a street, car park, park, green space or harbourside area in Bristol or look at and discuss problems that have already been reported.</h2>
```

Comment:
I'm not sure why this change has been made. Presumably Bristol's current look is what they have asked for, and it seems to remain consistent as a `p` - ideally nothing should visually change as a result of this PR, I assume?